TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/Compiler.php

```diff
@@ -141,6 +141,9 @@ public function getProxyClass($fullClassName)
         if (in_array(substr($fullClassName, 0, 14), $this->blacklistedSubPackages)) {
             return false;
         }
+        if ($classReflection->isFinal() && preg_match('/^\s?\*\s?\@Annotation\s/m', $classReflection->getDocComment()) === 1) {
```

Comment:
At least it feels like this rule is very specific and therefore dirty. I definitely prefer the proxy false solution then, also this might have impact on other peoples packages then...